django_project/feti/static/js/scripts/share.js

```diff
@@ -68,6 +68,19 @@ define(['backbone', 'jquery', 'underscore'], function (Backbone, $, _) {
                 windowReference.location = twitter_intent;
             });
         };
+        
+        this.shareToFacebook = function () {
+            // get url
+            var host = Backbone.history.location.host;
+            var windowReference = window.open('', '_blank', 'location=yes,height=570,width=520,scrollbars=yes,status=yes');
+
+            _generateURL(function (data) {
+                var facebook_sharer = 'http://www.facebook.com/sharer/sharer.php?s=100&p' +
```

Comment:
IIRC, this won't work if we have ssl installed.

Reply:
@myarjunar

Reply:
will changing it to `https` do the trick?